### ISSUE TITLE
jb update on an existing jsonnetfile.json (but created with master) fails

### ISSUE THREAD

**Issue by davidovich** [jsonnet-bundler/jsonnet-bundler#74]
I believe the upgrade path of the new jb is broken for old (v0.2.0) jb users with operating on an inited jsonnetfile.json from a master jb.

To reproduce: install jb at latest (v0.2.0):

`GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb@latest`

clone tanka (https://github.com/grafana/tanka)

cd examples/prom-graphana

```bash
david$ jb update
jb: error: failed to install packages: downloading: failed to create tmp dir: stat vendor/.tmp: no such file or directory
```

works when I update jb to master.

**Comment by sh0rez** [comment]
Yeah, master contains backwards incompatible changes

**Comment by sh0rez** [comment]
This is caused by the same issue as #75 where we changed the schema and older / newer versions can't keep up.

When implemented, #76 will provide a proper fix for this.
